### DISCUSSION TITLE
Consulta: Factura electronica en Punto de Venta

### DISCUSSION THREAD

**Discussion by Walter1109** [OdooAR/odoo-argentina-doc#17] [Q&A]
Hola, soy nuevo en Odoo, y estoy utilizando Odoo 14 CE, Localizacion Codize, en Ubuntu 20.04 lts.
Quisiera saber si se puede y como? generar una factura electronica desde Punto de Venta en formato de ticket.

**Comment by ibuioli** [comment]
Buenas! Es posible, aunque cada localización tiene sus temas en Community, son parecidas pero puede diferir (en EE ya está soportado de forma nativa). Para que te funcione necesitas los paquetes de regaby: https://github.com/regaby/odoo-custom concretamente el [l10n_ar_pos_einvoice](https://github.com/regaby/odoo-custom/tree/14.0/l10n_ar_pos_einvoice) y el [l10n_ar_pos_einvoice_ticket](https://github.com/regaby/odoo-custom/tree/14.0/l10n_ar_pos_einvoice_ticket)
Despues hay uno que dice moldeo_fix que puede ser necesario, debido a que para la de Codize nos basamos en la de Moldeo.

**Comment by Walter1109** [comment]
Muchas gracias Ignacio.
